tajo-algebra/src/main/java/org/apache/tajo/algebra/WindowSpec.java

```diff
@@ -128,12 +128,14 @@ public boolean equals(Object obj) {
   public static enum WindowFrameStartBoundType {
     UNBOUNDED_PRECEDING,
     CURRENT_ROW,
-    PRECEDING
+    PRECEDING,
+    FOLLOWING
   }
```

Comment:
Right. I hope that you read a couple of comments in the thread of the link. As I commented there, there is only one rule that the end bound cannot precede the start bound. So, naturally, the start bound cannot be unbounded following. 
I think that we can make some codes simpler with the unified bound type.